ci-operator/step-registry/aro-hcp/lease/release/aro-hcp-lease-release-commands.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck disable=SC1091
+source "$LEASE_PROXY_CLIENT_SH"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)